__tests__/app.test.js

```diff
@@ -85,3 +83,61 @@ describe("GET review by id, api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("Update review by review_id, PATCH /api/reviews/:review_id", () => {
+  test(`Status 201 - Returns with the updated review, when provided with a review_id and a new Vote object`, () => {
```

Comment:
incorrect status code. 201 is "created" you've not created anything. Stick to 200